fuzzing/run.sh

```diff
@@ -14,6 +14,8 @@ RESULTS=_results
 mkdir -p "$RESULTS"
 mkdir -p "$EXAMPLES"
 
+cp -f "../tests/data/*" "$EXAMPLES"
```

Comment:
This fails for me because it is trying to access a literal file "*" in data:

```
cp: cannot stat '../tests/data/*': No such file or directory
```

Without the quotation marks or with `"../tests/data/"*` it works